presto-native-execution/presto_cpp/main/types/tests/FunctionMetadataTest.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "presto_cpp/main/types/FunctionMetadata.h"
+#include <boost/filesystem.hpp>
+#include <gtest/gtest.h>
+#include "presto_cpp/main/common/tests/test_json.h"
+#include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+#include "velox/functions/prestosql/window/WindowFunctionsRegistration.h"
+
+namespace fs = boost::filesystem;
+
+using namespace facebook::velox;
+using namespace facebook::presto;
+
+using json = nlohmann::json;
+
+static const std::string kPrestoDefaultPrefix = "presto.default.";
+static const std::string kDefaultSchema = "default";
+
+namespace {
+std::string getDataPath(const std::string& fileName) {
```

Comment:
There should be a common utility for this logic as it might be required in all tests that need verififcation with hard-coded json. Please can you check if it exists already or make this a utility function.

Reply:
Thanks @aditi-pandit, I agree. There isn't a common utility function for this but I was planning to extract this into a utility function in a separate PR, based on suggestions from the previous review. Could you please confirm if I can add it back to this PR?